components/tools/OmeroWeb/omeroweb/webclient/show.py

```diff
@@ -685,7 +685,8 @@ def paths_to_object(conn, experimenter_id=None, project_id=None,
             select coalesce(sowner.id, plowner.id, aowner.id, wsowner.id),
                    slink.parent.id,
                    plate.id,
-                   acquisition.id
+                   acquisition.id,
+                   well.id
             from WellSample wellsample
             left outer join wellsample.details.owner wsowner
```

Comment:
Unrelated to this PR, but why are `wsowner` and `aowner` the same: `wellsample.details.owner`?